frontend/packages/integration-tests-cypress/support/index.ts

```diff
@@ -59,7 +59,7 @@ Cypress.Commands.add('testA11y', (target: string) => {
 export const checkErrors = () =>
   cy.window().then((win) => {
     if (win.windowError) {
-      throw new Error(`window/js runtime error detected: ${win.windowError}`);
+      throw new Error(`window error - unhandled promise rejection: ${win.windowError.reason}`);
```

Comment:
We lose the stack trace if you do this. It would be better. We need to print that out somehow or we won't be able to debug these errors.

Reply:
If `win.windowError` is always a `PromiseRejectionEvent` then it doesn't have a `e.stack` rather it has a `PromiseRejectionEvent.reason`.  If I throw a new Error, it's stack with be to the 'checkErrors' method, not the original error.

Reply:
The code you pasted shows it's not always `PromiseRejectionEvent` :) It can be `window.onerror`, too.

Reply:
resolved, pls see latest code